src/docs/08_concepts.adoc

```diff
@@ -49,19 +49,82 @@ on this list.
 image:08-Crosscutting-Concepts-Structure-EN.png["Possible topics for crosscutting concepts"]
 ****
 
+////
+
+=== Domain concepts and model
+
+
+[plantuml,"Sequence diagram",png]
+----
+@startuml
+User -- Route
+User -- User : hasFriend
+Route "1" *--> "1..*" Waypoint
+Route "1" *--> "0..*" UserComment
+Waypoint o-- "0..*" Image
+Waypoint o-- "0..*" Video
+@enduml
+----
+
+A *route* must contain a name, optionally it can have a description.
+Its composed by at least one waypoint and may or not have a list of comments and media.
+
+The *Waypoint* must have latitude and longitude. They can have as optional properties a name, altitude and a physical address.
+
+A *userComment* must contain the text of the comment, the time it was published and the author, being the last one a IRI to a pod profile shape.
+
+And finally, an *Image* or a *Video* must have an IRI to te resource, another IRI to the pod profile of the author and the time it was published
```

Comment:
Do yo mean URI instead of IRI? please review that if you don´t mind

Reply:
I meant IRI. Dont know if its wrong but that is what Labra uses in the shex definitios [here](http://shex.io/)

Reply:
Ok, LGTM